builtin/types/dpos/dpos.proto

```diff
@@ -3,17 +3,17 @@ syntax = "proto3";
 import "github.com/loomnetwork/go-loom/types/types.proto";
 
 message Params {
-    uint64 witness_count = 1;
+    uint64 validator_count = 1;
     uint64 vote_allocation = 2;
     int64 election_cycle_length = 3;
     uint64 min_power_fraction = 4;
     Address coin_contract_address = 5;
-    uint64 witness_salary = 6;
+    uint64 validator_salary = 6;
 }
 
 message State {
     Params params = 1;
-    repeated Witness witnesses = 2;
+    repeated Validator validators = 2;
```

Comment:
This isn't a backwards compatible change since `Witness` and `Validator` have different field layout. I'll let @mattkanwisher decide if we care about that with this contract, if we do may be best to create a v2 of this contract and leave the old one unchanged, then figure out how to migrate from v1 to v2.

Reply:
I think thats fine